### PR TITLE
#24 Added ZipForm, Genres, and Artists component 

### PR DESCRIPTION
**Pull request by Miguel-Villanueva**
***** **Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines. For the guidelines check the README.
- [ ] Tests for the changes have been added (for bug fixes / features). As we discussed this is optional but encouraged.

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
This adds the following features: allows user to input postal code, allows user to choose from a form of genres of concerts happening by postal code, allows user to choose from a form of artists preforming at the postal code within the set of user selected genres (displays the user selected list for now but will use for spotify api input in the future) 

* **What is the current behavior?** (You can also link to an open issue here)
Current behavior displays the different pages of the application

* **What is the new behavior (if this is a feature change)?**
ZipForm displays a postal code user input
Genres displays a form of concert genres happening in inputted user postal code
Artists displays a form of artists preforming in inputted user postal code and user selected genres

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
Yes, changes are included in readme: 

- Must create .env file in root directory with the line 
      REACT_APP_API_KEY=

- Must run "npm install dotenv"

* **Other information**:****

### CODE REVIEW

**Review by berabulut — APPROVED** [review]
Works good on my local. Great job!
https://prnt.sc/tmdryt

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_API_KEY=
```
Comment (csjoblinksreddit) [inline .env]: Can we add this file to the .gitignore. That way we make sure that the .env file that contains the key will never be pushed to github.

After seeing the change down below,  I think the its added to the incorrect .gitignore file.

File: src/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+.env
```
Comment (csjoblinksreddit) [inline src/.gitignore]: Can we delete this file and keep one .gitignore for the frontend. The current .gitignore is located in the main directory.

File: src/components/Genres.css [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  
+  @keyframes App-logo-spin {
+    from {
+      transform: rotate(0deg);
+    }
+    to {
+      transform: rotate(360deg);
+    }
+  }
+
```
Comment (csjoblinksreddit) [inline src/components/Genres.css]: Can we add the css and js files into a folder names Genres. That way we keep our project looking clean with the css and js files together.

**Review by csjoblinksreddit — CHANGES_REQUESTED** [review]

**Comment by csjoblinksreddit** [comment]
Hey Miguel one other thing is that I couldnt find the ticket you are referring to. AT-5 does not exists. Perhaps you meant [AT-24](https://afternoonteamreddit.atlassian.net/browse/AT-24)

File: .env (hunk 1 above)
Comment (Miguel-Villanueva) [reply .env]: Added

File: src/.gitignore (hunk 2 above)
Comment (Miguel-Villanueva) [reply src/.gitignore]: Deleted unnecessary gitignore

File: src/components/Genres.css (hunk 3 above)
Comment (Miguel-Villanueva) [reply src/components/Genres.css]: added folder

**Review by csjoblinksreddit — CHANGES_REQUESTED** [review]
This looks great! The last thing we have to do is remove the .env file from your branch so it doesnt get merged. Ill show you how to do that tomorrow.

File: package.json [hunk 4]
```diff
@@ -6,6 +6,10 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "dotenv": "^8.2.0",
+    "react": "^16.13.1",
```
Comment (csjoblinksreddit) [inline package.json]: I dont think we need react, react-dom,and react-scripts as these are already added below.

Comment (Miguel-Villanueva) [reply package.json]: deleted duplicates

**Comment by csjoblinksreddit** [comment]
I may be running this incorrectly, but is there a route associated with these changes? I thought this was supposed to be routed when I click on the concert playlist page?

**Pull request closed without merging**